### DISCUSSION TITLE
Did the UK MSFS update break the mod again?

### DISCUSSION THREAD

**Discussion by Jujmental** [flybywiresim/aircraft#3579] [General]
I installed another fresh version of the mod, just in case it needed updating but my A320 now takes off like a helicopter - full nose down attitude despite the climb indicator showing positive rate. I'm assuming ASOBO broke your fine mod again.

KR David

**Comment by ChruutvoLuzi** [comment]
yes, download the latest dev, it has been fixed an hour ago or so

**Reply by Jujmental** [reply]
Thank you!
